src/main/java/net/minestom/server/entity/Metadata.java

```diff
@@ -151,10 +151,6 @@ public static Entry<RegistryKey<WolfSoundVariant>> WolfSoundVariant(@NotNull Reg
         return new MetadataImpl.EntryImpl<>(TYPE_WOLF_SOUND_VARIANT, value, WolfSoundVariant.NETWORK_TYPE);
     }
 
-    public static Entry<FrogMeta.Variant> FrogVariant(@NotNull FrogMeta.Variant value) {
```

Comment:
Why is this removed? This is still a valid entity meta type. It just needs to be registry bound instead of enum bound